_posts/2015-04-24-FFI-and-Rust.md

```diff
@@ -0,0 +1,285 @@
+---
+layout: post
+title: "FFI and Rust"
+author: Alex Crichton
+description: "Zero-cost and safe FFI in Rust"
+---
+
+Rust's quest for world domination was never destined to happen overnight, so
+Rust needs to be able to interoperate with the existing world just as easily as
+it talks to itself. In particular, **Rust makes it easy to communicate with C
```

Comment:
Maybe s/in particular/for this reason/? I found "in particular" odd here because the structure of the sentence that follows is not parallel to the one before -- that is, this is not a refinement of a more general claim. Before you said "rust needs to be able to do this". I would expect something like, "in particular, rust needs to be able to call C functions" or something, rather than "in particular, rust makes it easy to call C functions".